### PR TITLE
loopd: make term and quote calls readable

### PR DESCRIPTION
**Pull request by hieblmi**
Removing loop permissions from the quote and term calls.

### CODE REVIEW

File: loopd/perms/perms.go [hunk 1]
```diff
@@ -34,30 +34,18 @@ var RequiredPermissions = map[string][]bakery.Op{
...
-	}, {
-		Entity: "loop",
-		Action: "in",
 	}},
 	"/looprpc.SwapClient/GetLoopInQuote": {{
 		Entity: "swap",
 		Action: "read",
-	}, {
-		Entity: "loop",
-		Action: "in",
```
Comment (GeorgeTsagk) [inline loopd/perms/perms.go:60]: I wonder if there was a strong rationale on placing these here?

**Review by GeorgeTsagk — APPROVED** [review]
LGTM 🕸️

File: loopd/perms/perms.go (hunk 1 above)
Comment (hieblmi) [reply loopd/perms/perms.go:60]: See my comment here, https://github.com/lightninglabs/terminal-web/issues/662#issuecomment-1594284365. We have to find out :-)

**Review by alexbosworth — APPROVED** [review]

**Comment by hieblmi** [comment]
Instead of removing the `loop:in`/`loop:out` perms we preserve them and add these to the client side macaroon permissions. `loop:in`/`loop:out` stay in place here to be able to scope for only one of the directions in the client macaroon.

**Pull request closed without merging**